### PR TITLE
Solo Project 1: Building with Backgrounds and Gradients

### PR DESCRIPTION
**Pull request by appiersign**
### APPLE WEBSITE CLONE: First Solo Project
In this Project
- I added some semantic HTML tags to the index page
- I fixed the background gradient of the nav bar
- I added styling to nav search box
- I added linter
- I added a descriptive PR 
- fixed invisible h tag texts in header
- replaced divs with img tags wrapped in a
- changed #products div selector to class

### CODE REVIEW

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Solomon!  
### Your did a good job! :+1: 
Here list of things you need to change before this pull request can be approved:
* [ ] Check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). Now it shows errors in your project.
* [ ] `hero-texts` is not displaying right now and navbar is not centered, you need to fix that:
image
* [ ] `products` should contain four elements which are links (to another articles or `#`) ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/)). Right now you have only `<div>`  elements, please add links to them.

[OPTIONAL]
* It's a good practice to use the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links
* It's a good practice to use the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document
### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Comment by pelzolga123** [comment]
## Great work :hammer: 
Still a few more changes left:
* [ ] Check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). It still shows errors in your project.
* [ ] Header image is not centered, that's why page looks a bit messed up. It can be easily fixed, if you remove `min-width: 1440px` from `header`.

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

File: style.css [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+  text-decoration: none;
+}
+
+#main-nav {
+  display: flex;
+  align-items: flex-end;
+  width: 980px;
+  height: 114px;
+}
+
```
Comment (ebukaume) [inline style.css]: Why are you making this width to be 980px? Consider using a value in `%` here so it adjusts to different screen sizes.

**Review by ebukaume — CHANGES_REQUESTED** [review]
Hi @appiersign :wave: 

You are doing great so far. However, it stills a bit weird on my system.
image

_See below for more..._

**Kindly submit another code review request after making this fix**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :clap: 

Congratulations **Solomon** for making all the changes requested by previous reviewers :slightly_smiling_face: . You can now merge to master and submit project completion form.

Good luck with your next project.
Happy coding. :computer:

**Pull request merged**